canvas_cli/main.py

```diff
@@ -51,6 +58,37 @@ def get_or_create_config_file() -> Path:
     return config_path
 
 
+@app.command()
+def emit(
+    event_fixture: str,
+    plugin_runner_port: Annotated[
+        str, typer.Option(help="Port of your locally running plugin runner")
+    ] = "50051",
+) -> None:
+    """
+    Grab an event from a fixture file and send it your locally running plugin-runner process.
+    Any resultant effects will be printed.
+
+    Valid fixture files are newline-delimited JSON, with each containing the keys `EventType`, `target`, and `context`. Some fixture files are included in the canvas-plugins repo.
+    """
+    # Grab a random event from the fixture file ndjson
+    lines = open(event_fixture).read().splitlines()
+    myline = random.choice(lines)
+    event_data = json.loads(myline)
```

Comment:
Why do we want to make this random? If I am a plugin developer and my plugin has a specific event that it responds to, I think there would be more value in being able to specify the event that is sent.

Reply:
Similar to randomizing the test order in pytest, I can see value in ensuring your plugin behaves as expected in a variety of scenarios. If a developer wants to test against a fixed event, they are able to create their own ndjson with a single line and pass that file path in as the CLI arg.